tests/mock_tables/asic0/appl_db.json

```diff
@@ -86,5 +86,17 @@
   "INTF_TABLE:PortChannel01:10.10.0.1/31": {
     "scope": "global",
     "family": "IPv4"
+  },
+  "NEIGH_TABLE:Ethernet0:10.0.0.0": {
```

Comment:
>10.0.0.0 [](start = 25, length = 8)

Add some mock data for IPv6
Add some mock data for PortChannel
Will new mock data need more test cases?

Reply:
Added data for IPv6 and Port Channel. Added one more test case to make sure that eth0 of namespace, which is a part of docker0 bridge, is not used, for multi-asic platform.